src/i18n.ts

```diff
@@ -1,9 +1,12 @@
 import { createI18n } from 'vue-i18n';
+import { nextTick } from 'vue';
 import messages from '@/locales';
```

Comment:
I think lazy loading doesn't work, the translations should have their own files in the bundle but i don't see it here:
https://bafybeigvveu2kmjcbu5s2i2yuedwswbfmpalctf2t5vd5q6nunjk3mhuju.on.fleek.co/modern-report.html
It's same as before see here: https://snapshot.org/modern-report.html
I think the issue come from this `import messages`, i assume it add all translation in the app bundle.